src/server/services/Actions.ts

```diff
@@ -37,12 +38,16 @@ export default class Actions {
       waitReady().then(() => {
         this.account = keyring.addFromMnemonic(mnemonic);
 
-        setInterval(() => {
-          // We do want the following to just start and run
-          // eslint-disable-next-line @typescript-eslint/no-floating-promises
-          // TODO: Adding a subscription would be better but the server supports on http for now
-          this.updateFaucetBalance().catch(console.error);
-        }, balancePollIntervalMs);
+        // We do want the following to just start and run
+        // eslint-disable-next-line @typescript-eslint/no-floating-promises
+        // TODO: Adding a subscription would be better but the server supports on http for now
+        const updateFaucetBalance = (log = false) => {
+          if (log) logger.info('Fetched faucet balance 💰');
```

Comment:
This log line is misleading because it says "Fetched" before the call is made. What about "Fetch`ing`"?